src/main/java/org/zalando/planb/provider/AuthorizeController.java

```diff
@@ -49,23 +99,27 @@
     @Autowired
     private CassandraAuthorizationCodeService cassandraAuthorizationCodeService;
 
+    @Autowired
+    private CassandraConsentService cassandraConsentService;
```

Comment:
It would we nice to extract an interface and hide the Cassandra implementation detail